dymos/phase/phase.py

```diff
@@ -619,6 +619,15 @@ def add_control(self, name, order=_unspecified, units=_unspecified, desc=_unspec
             self.control_options[name] = ControlOptionsDictionary()
             self.control_options[name]['name'] = name
 
+        # if continuity, rate_continuity, or rate2_continuity are not specified, default to False for cases when control_type is 'polynomial'
+        if control_type == 'polynomial':
+            if continuity is _unspecified:
```

Comment:
There's no such thing as continuity here anyway right?  To specify it is wrong.  I think you could cover the polynomial case more concisely with `continuity = rate_continuity = rate2_continuity = False` if the control_type == 'poylnomial'

Reply:
Got it, updated accordingly!